lib/publiccloud/instance.pm

```diff
@@ -976,4 +976,46 @@ sub wait_for_state {
     die("The instance state is not '$state' but '$current' instead.");
 }
 
+sub check_selinux_pc {
+    # Check SELinux enforced VMs on publiccloud
+    my ($self) = @_;
+    my $log_avc = "selinux_avc_audit.txt";
+    my $ssh_opts = "-t -o ControlPath=none";
+    die("Not a Public cloud environment.") unless (is_public_cloud);
+    return unless (is_sle_micro || is_sle);
+
+    my $instance = $self;    # $self->{my_instance};
+    my $out;
+
+    # check SElinux status:
+    $out = $instance->ssh_script_output("cat /proc/cmdline", ssh_opts => $ssh_opts);
```

Comment:
I already explained why checking `/proc/cmdline` is wrong.  Any image could have SELinux enabled and it's irrelevant if `selinux` is present in the kernel command line or not.

Reply:
I already explained this.  Running another ssh command to grab the content of a file that is completely unrelated to SELinux is a waste of time.